Bio/Blast/NCBIWWW.py

```diff
@@ -172,15 +172,14 @@ def qblast(program, database, sequence, url_base=NCBI_BLAST_URL,
     # will take longer thus at least 70s with delay. Therefore,
     # start with 20s delay, thereafter once a minute.
     delay = 20  # seconds
-    previous = time.time()
```

Comment:
You should keep this reset of the timer - it represents the moment after submitting the request, after which we should wait before checking on progress.